src/components/PostDetailsContent.tsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { NewCommentForm } from './NewCommentForm';
+
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import {
+  deleteCommentAsync,
+  loadCommentsAsync,
+} from '../features/commentsSlice';
+
+export const PostDetailsContent = () => {
+  const [visible, setVisible] = useState(false);
+  const post = useAppSelector(state => state.selectedPost);
+  const { comments, isLoading, hasError } = useAppSelector(
+    state => state.comments,
+  );
+
+  const dispatch = useAppDispatch();
+
+  function loadComments() {
+    setVisible(false);
+    if (post) {
+      dispatch(loadCommentsAsync(post.id));
+    }
+  }
+
+  useEffect(loadComments, [post?.id]);
+
+  const deleteComment = async (commentId: number) => {
+    dispatch(deleteCommentAsync(commentId));
```

Comment:
- Use one style for your functions
- If you don't use function only in useEffect, you don't need to move it outside it